tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -88,6 +88,40 @@ Synthetics:
   [13..16): Int => _root_.scala.reflect.ClassTag.Int.
 
 
+scalameta/semanticdb-integration/src/main/scala/example/Overrides.scala
+-----------------------------------------------------------------------
+Language:
+Scala212
+
+Names:
+[8..15): example <= _root_.example.
+[23..24): A <= _root_.example.A#
+[31..34): foo <= _root_.example.A#foo()I.
+[36..39): Int => _root_.scala.Int#
+[48..49): B <= _root_.example.B#
+[49..49): ε <= _root_.example.B#`<init>`()V.
+[60..61): A => _root_.example.A#
+[62..62): ε => _root_.java.lang.Object#`<init>`()V.
+[68..71): foo <= _root_.example.B#foo()I.
+[73..76): Int => _root_.scala.Int#
+
+Symbols:
+_root_.example. => package example.{+4 members}
+_root_.example.A# => trait A.{+1 members}
+_root_.example.A#foo()I. => abstract def foo: Int
+  [0..3): Int => _root_.scala.Int#
+_root_.example.B# => class B
+_root_.example.B#`<init>`()V. => primaryctor <init>: (): B
+  [4..5): B => _root_.example.B#
+_root_.example.B#foo()I. => def foo: Int
+  [0..3): Int => _root_.scala.Int#.{override _root_.example.A#foo()I.}
```

Comment:
^

Reply:
Hmm, is `scala.Int` overriding `A#foo()` or am I misunderstanding the denotation syntax? I would expect the syntax to be something like this instead
```scala
_root_.example.B#foo()I. => def foo: Int
  [0..3): Int => _root_.scala.Int#.
  override _root_.example.A#foo()I.
```

wdyt?

Reply:
```scala
trait A { def foo: Int }
class B() extends A { def foo: Int = 2 }
```
yep, I think it's not the correct symbol.